service-registry/src/main/java/org/apache/servicecomb/serviceregistry/client/LocalServiceRegistryClientImpl.java

```diff
@@ -118,14 +118,19 @@ private void initFromData(Map<String, Object> data) {
         String appId = (String) serviceConfig.get("appid");
         String version = (String) serviceConfig.get("version");
         String serviceId = (String) serviceConfig.get("id");
-
+        @SuppressWarnings("unchecked")
```

Comment:
Can you add some unit tests in TestLocalServiceRegistry?

Reply:
added